.github/workflows/main.yml

```diff
@@ -1,15 +1,68 @@
-name: Ubuntu
+name: CI
 
-on: [push]
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
 
 jobs:
-  build:
-
-    runs-on: ubuntu-18.04
+  test:
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [3.6]
+        python-version: [pypy3, 2.7, 3.5, 3.6, 3.7, 3.8]
+        os: [macos-latest, ubuntu-latest, windows-latest]
```

Comment:
I'd rather use fixed versions to avoid things breaking unexpectedly. For Ubuntu, I think we should test both 18.04 and 20.04.